perun-core/src/main/java/cz/metacentrum/perun/core/impl/modules/attributes/urn_perun_group_attribute_def_def_unixGID_namespace.java

```diff
@@ -118,7 +118,17 @@ public Attribute fillAttribute(PerunSessionImpl sess, Group group, AttributeDefi
 	}
 
 	@Override
-	public void checkAttributeSemantics(PerunSessionImpl sess, Group group, Attribute attribute) throws InternalErrorException, WrongAttributeValueException, WrongReferenceAttributeValueException, WrongAttributeAssignmentException{
+	public void checkAttributeSyntax(PerunSessionImpl sess, Group group, Attribute attribute) throws WrongAttributeValueException, InternalErrorException {
+		//Check if GID is within allowed range
+		try {
+			sess.getPerunBl().getModulesUtilsBl().checkIfGIDIsWithinRange(sess, attribute);
+		} catch (WrongAttributeAssignmentException | AttributeNotExistsException e) {
```

Comment:
I am afraid that null is (in specific situation) allowed. So by syntax check it should be allowed always.

Reply:
To be within range is semantic problem here.